projects/control-service/cicd/.gitlab-ci.yml

```diff
@@ -157,6 +157,32 @@ control_service_publish_job_builder_image:
     changes:
       - projects/control-service/projects/helm_charts/pipelines-control-service/version.txt
 
+
+control_service_publish_job_builder_secure_image:
+  image: docker:19.03.15
```

Comment:
These fields (services, variables, stage, retry, etc.) are common for all Job Builder steps. So we can extract them into a base step.

Reply:
done